docs/parallelism.md

```diff
@@ -0,0 +1,31 @@
+# Limiting parallelism
+
+You can restrict the number of simultaneous workflow executions.
+
+## Controller level
+
+You can limit the total number of workflows that can execute at any one time in the [workflow controller ConfigMap](./workflow-controller-configmap.yaml).
+
+```yaml
+data:
+  parallelism: "10"
+```
+
+You can also limit the number of workflows that can execute in a single namespace.
+
+```yaml
+data:
+  namespaceParallelism: "4"
+```
+
+Workflows that are executing but restricted from running more nodes due to the other mechanisms will still count towards the parallelism limits.
+
+### Priority
+
+Workflows can have a `priority` set in their specification.
```

Comment:
```suggestion
You can set a `priority` on Workflows.
```

active voice, same as above

Reply:
Same problem as above.

Reply:
I've added a description of how to use this, and the missing information of default priority.